generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-delete-dialog.component.html

```diff
@@ -1,4 +1,4 @@
-<%_ var keyPrefix = angularAppName + '.' + entityTranslationKey + '.'; _%>
+<%_ var keyPrefix = angularAppName + '.' + entityTranslationKey; _%>
```

Comment:
I think its just that when we use it in the jhi-translate directive we already have a "." right after keyPrefix so it was causing two dots to be printed